config/backend-config/multitenant_workspaces.go

```diff
@@ -68,44 +69,33 @@ func (workspaceConfig *MultiTenantWorkspacesConfig) GetWorkspaceLibrariesForWork
 }
 
 // Get returns sources from the workspace
-func (workspaceConfig *MultiTenantWorkspacesConfig) Get(workspaces string) (ConfigT, bool) {
-	return workspaceConfig.getFromAPI(workspaces)
+func (workspaceConfig *MultiTenantWorkspacesConfig) Get(ctx context.Context, workspaces string) (ConfigT, error) {
+	return workspaceConfig.getFromAPI(ctx, workspaces)
 }
 
 // getFromApi gets the workspace config from api
-func (workspaceConfig *MultiTenantWorkspacesConfig) getFromAPI(workspaceArr string) (ConfigT, bool) {
+func (workspaceConfig *MultiTenantWorkspacesConfig) getFromAPI(
+	ctx context.Context, workspaceArr string,
+) (ConfigT, error) {
 	// added this to avoid unnecessary calls to backend config and log better until workspace IDs are not present
 	if workspaceArr == workspaceConfig.Token {
```

Comment:
minor: not related to this commit, but the name workspaceArr is confusing (i thought it referred to a list of workspaces). Is Arr short for array or refers to something else? Its type seems to be a string. if its something else, maybe we should rename it ?

Reply:
I think it refers to `array`. Feel free to open a PR to address the issue :+1: